_format/1.1/index.md

```diff
@@ -1962,6 +1988,10 @@ To do so, a server **MAY** define an internal mapping from query parameter names
 to profile URIs. The profile URI for a query parameter name in this mapping 
 **MUST NOT** change over time.
 
+> Note: the server may choose to map all query parameter names from the same 
```

Comment:
Not sure I follow... since every server got to define it's own mapping of parameters to profile urls even before we introduced the parameter vs. parameter family distinction, hasn't that always been the case?

Reply:
I misinterpreted this.

I thought this was referring to the case when multiple profiles overlap in which query parameter names they react to/define behavior for — say `page[special]` is defined tiwce. "server may choose" then made me think "but then how can the client know which profile's behavior is going to apply?"

Reply:
Ok, cool. Happy to clarify the text too if you think it's warranted.